src/network/UDPConnection.hpp

```diff
@@ -0,0 +1,39 @@
+/*
+** EPITECH PROJECT, 2024
+** RType
+** File description:
+** UDPConnection
+*/
+
+#ifndef UDP_CONNECTION_HPP_
+    #define UDP_CONNECTION_HPP_
+    #include <iostream>
```

Comment:
Also add a line between define and include for better readibility

Reply:
Done